.github/workflows/deploy-docs.yml

```
@@ -7,7 +7,10 @@
     paths:
       - "docs/**"
       - ".github/workflows/deploy-docs.yml" # define the concrete paths on which a change triggers this action, e.g. backend/**
-
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)